quipucords/api/serializers.py

```diff
@@ -11,9 +11,11 @@
 """Module for serializing all model object for database storage"""
 
 import os
+from django.db import transaction
 from django.utils.translation import ugettext as _
-from rest_framework.serializers import ModelSerializer, ValidationError
-from api.models import Credential, HostCredential
+from rest_framework.serializers import ModelSerializer, ValidationError, \
+    SlugRelatedField, PrimaryKeyRelatedField
```

Comment:
Another option for breaking up long import lines is

```
from rest_framework.serializers import (
    ModelSerializer,
    ValidationError,
    SlugRelatedField,
    PrimaryKeyRelatedField,
)
```
This is also a bit more extensible, creating smaller diffs in the future if more imports are added.